docs/reference/aggregations/bucket/variablewidthhistogram-aggregation.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[search-aggregations-bucket-variablewidthhistogram-aggregation]]
+=== Variable Width Histogram Aggregation
+
```

Comment:
I talked with @polyfractal and @giladgal about this PR and we'd like to mark it as `experimental[]` so we can make breaking changes to its API. It is different enough from what we've done in the past that we want a little time not subject to semver.

@jamesdorfman, there is nothing you have to do for this, but you'll see an issue tracking the experimental-nature of the agg pretty soon.